src/Sulu/Bundle/SearchBundle/DependencyInjection/Configuration.php

```diff
@@ -45,6 +45,15 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('website')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('indexes')
+                            ->useAttributeAsKey('key')
```

Comment:
Does it make sense to set this? If I read the [Symfony documentation](https://symfony.com/doc/3.2/components/config/definition.html#array-node-options) correctly, the `array_values` call in our code would be unnecessary if we just omit that, because when the config is evaluated it will already be returned as simple array. Not totally sure if overriding as you've suggested in this PR would still work though.

Reply:
just tried, overriding doesn't work then